mafia/data/garbo_settings.json

```diff
@@ -51,6 +51,11 @@
     "type": "boolean",
     "description": "**WARNING: Experimental** Don't show user confirm dialogs, instead automatically select yes/no in a way that will allow garbo to continue executing. Useful for scripting/headless. Risky and potentially destructive."
   },
+  {
+    "name": "garbo_autoUserConfirm_embezzlerInvocationsCount",
```

Comment:
I think this should be changed to `garbo_autoUserConfirm_embezzlerInvocationsThreshold`.